be/src/exec/vectorized/chunks_sorter_topn.cpp

```diff
@@ -362,19 +362,27 @@ void ChunksSorterTopn::_merge_sort_common(ChunkPtr& big_chunk, DataSegments& seg
     }
     ChunkPtr right_chunk = big_chunk->clone_empty(permutation_second.size());
     append_by_permutation(right_chunk.get(), right_chunks, permutation_second);
+    Columns right_columns;
+    for (auto expr : *_sort_exprs) {
+        auto maybe_column = expr->evaluate(right_chunk.get());
+        CHECK(maybe_column.ok());
```

Comment:
Please use DCHECK to avoid crash because of one bad case.

Reply:
done. Make this function return a status.